README.ubi8mini.md

```diff
@@ -0,0 +1,28 @@
+# OpenDXL Broker
+
+## Overview
+
+A README related to building a Red Hat Universal Base Image 8 compatible version of the OpenDXL broker.
+
+## Documentation
+Changes to the broker for OpenSSL 1.1.0 and provide a mechanism for OpenDXL broker compatibility with Red Hat Universal Base Image 8
```

Comment:
It seems the OpenSSL 1.1.0 changes are general and not specific to the Red Hat Dockerfile, etc. Therefore, it doesn't seem like they should appear in this file.

Reply:
ok. I will remove from this file